index.css

```diff
@@ -0,0 +1,107 @@
+:root {
+    --primary-color: #002D62;
+    --secondary-color: white;
+    --primary-shade-color:  #194271; 
+    --font-family: sans-serif;
+    font-size: 16px; 
+}
+
+body {
+    background-color: var(--primary-color) ;
+}
+
+.categories {
+    display: grid;
+    grid-template-columns: repeat(3,auto) ; /* create a 3 column layout of equal width **/
+    grid-auto-rows: auto;
+    grid-auto-columns: auto;
+    grid-gap: 5px;
+    
+    justify-content: center;
+    align-content: center;
+    grid-auto-flow: column;
+    padding: 0;
+    margin: 0;
+   margin-top: 10%;
+}
+
+.question-container {
+    width: 100px;
+    color: var(--secondary-color);
+}
+
+.question-container button {
+    width: 100%;
+    color: var(--secondary-color);
+    border-radius: 25px;
+}
+
+.text-align-center {
+  text-align: center;
+} 
+
+.center {
+  margin: auto;
+}
+
+.heading  {
+  font-size: 30px;

```

Comment:
Use rem here, that way it can scale as per users' preferences, which makes for a good user experience.